qual_id/pattern.py

```diff
@@ -42,6 +43,7 @@ class Pattern:
         "book": Book(),
         "brand": Brand(),
         "celebrity": Celebrity(),
+        "chemicalelement": ChemicalElement(),
```

Comment:
```suggestion
        "element": ChemicalElement(),
```
I think 'element' is descriptive enough. It will make it easier as a url parameter too 🙂